README.md

```diff
@@ -37,7 +37,9 @@ Triton Inference Server.
 help you automatically find the optimal settings for
 [Dynamic Batching](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#dynamic-batcher) and
 [Instance Group](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#instance-groups)
-parameters of your model configuration. Using
+parameters of your model configuration and utilizes 
+[Performance Analyzer](https://github.com/triton-inference-server/server/blob/main/docs/perf_analyzer.md) 
+to find optimal settings for concurrency and batch size. Using
```

Comment:
it isn't finding optimal concurrency/batch size, it is using that stimulus to help figure out which model configuration operates best under those configurations.

"and utilizes PA to test the model with different concurrency and batch sizes of requests

Reply:
updated